autoscale/src/main/java/com/sequenceiq/periscope/cache/DependentHostGroupsCache.java

```diff
@@ -0,0 +1,32 @@
+package com.sequenceiq.periscope.cache;
+
+import java.util.concurrent.TimeUnit;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.cache.common.AbstractCacheDefinition;
+
+@Component
+public class DependentHostGroupsCache extends AbstractCacheDefinition {
+
+    public static final String DEPENDENT_HOST_GROUPS_CACHE = "dependentHostGroupsCache";
+
+    private static final Long MAX_ENTRIES = 500L;
```

Comment:
Make this configurable in Periscope configs ?

Reply:
Updated this